HDDS-4485. [DOC] add the authentication rules of the Ozone Ranger. (#1603)
apache/ozone: hadoop-hdds/docs/content/security/SecurityWithRanger.md

Apache Ranger™ is a framework to enable, monitor and manage comprehensive data
security across the Hadoop platform. Apache Ranger has supported Ozone authentication 

> since version 2.0. However, due to some bugs in 2.0, we prefer to use Apache Ranger 
> 2.1 and later version.

Reviewer: ```suggestion
since version 2.0. However, due to some bugs in 2.0, Apache Ranger 
2.1 and later versions are recommended.
```

--------|------------------------------------------------------------
ozone.acl.enabled         | true
ozone.acl.authorizer.class| org.apache.ranger.authorization.ozone.authorizer.RangerOzoneAuthorizer

> The ranger permissions corresponding to Ozone operation are as follows:

Reviewer: ```suggestion
The ranger permissions corresponding to the Ozone operations are as follows:
```
